test/integration/integration_test.go

```diff
@@ -193,7 +193,7 @@ func FeatureContext(s *godog.Suite) {
 		variableShouldNotBeEmpty)
 
 	// steps for rollout check
-	s.Step(`^services? "([^"]*)" rollout successfully$`,
+	s.Step(`^services? "([^"]*)" rollout successfully in "(\d+)" seconds$`,
 		minishift.rolloutServicesSuccessfully)
```

Comment:
The current implementation does not look clean. It seems like we are sneaking the timeout into the existing implementation. I would suggest a change in function name e.g. rolloutServicesWithTimeout. Ideally, there should be two kinds of rollouts i.e. one with the timeout and one without the timeout.

Reply:
@LalatenduMohanty Will add it, however I am not sure whether we want to really use.